fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/SheetHorizontalItemView.kt

```diff
@@ -64,12 +69,18 @@ class SheetHorizontalItemView: TemplateView {
             mainContainer.setOnClickListener {
                 onSheetItemClickListener?.onSheetItemClick(mSheetItem!!)
             }
+
+        mainContainer.isEnabled = !disabled
+        sheetItemTitle.isEnabled = !disabled
```

Comment:
Curious? Why do we need disable on all the fields?

Reply:
We need to disable on main container and sheetItemTitle for sure. mainContainer's disable makes it non clickable whereas sheetItemTitle's disable updates textColor. I added other disables just in case its needed to some disable functionality. Please advise if we should remove them.